ursh/indico_ursh/controllers.py

```diff
@@ -41,25 +42,33 @@ def _resolve_full_url(original_url):
     @staticmethod
```

Comment:
I would make this simply a standalone function in `util.py` instead of having it a static method here